network-api/networkapi/buyersguide/views.py

```diff
@@ -1,23 +1,88 @@
+from django.core.exceptions import ObjectDoesNotExist, ValidationError
+from django.db import Error
 from django.shortcuts import render
-from networkapi.buyersguide.models import Product
 from django.contrib.auth.decorators import login_required
 from django.conf import settings
+from django.views.decorators.csrf import csrf_protect
+from rest_framework.parsers import JSONParser
+from rest_framework.decorators import api_view, parser_classes, throttle_classes, permission_classes
+from rest_framework.response import Response
+from rest_framework.permissions import AllowAny
+
+from networkapi.buyersguide.models import Product, BooleanVote, RangeVote
+from networkapi.buyersguide.throttle import UserVoteRateThrottle, TestUserVoteRateThrottle
+
+vote_throttle_class = UserVoteRateThrottle if not settings.TESTING else TestUserVoteRateThrottle
 
 
 # Login required so we can continue to develop this and merge into master without the public seeing it.
 # Remove this when we're ready to launch.
 @login_required
 def buyersguide_home(request):
     products = Product.objects.all()
+    products = [p.to_dict() for p in products]
```

Comment:
just to prevent weirdness, `products = [p.to_dict() for p in Product.objects.all()]` and then just remove L22 above it?